app/src/main/java/org/oppia/app/databinding/ViewBindingAdapter.kt

```diff
@@ -39,3 +39,29 @@ fun setFlashingAnimation(view: View, isFlashing: Boolean) {
     view.alpha = 0f
   }
 }
+
+@BindingAdapter(
+  "app:hasRotationAnimation",
```

Comment:
This attribute does not make sense to me, why do we need this?
Because if we are passing both `app:isRotationClockwise ` and `app:rotationAngle ` then it will work or else not.

Reply:
updated